content/docs/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Documentation
+linkTitle: Docs
+params:
+  redirect: /docs/latest 301!
```

Comment:
Is this Hugo's feature or our theme's ?

Reply:
It's a custom layout (to be upstreamed to Docsy at some point).